com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Depth-of-Field.md

```diff
@@ -9,8 +9,7 @@ The Depth Of Field component applies a depth of field effect, which simulates th
 1. In the Scene or Hierarchy view, select a GameObject that contains a Volume component to view it in the Inspector.
 2. In the Inspector, navigate to **Add Override > Post-processing** and click on **Depth Of Field**. HDRP now applies **Depth Of Field** to any Camera this Volume affects.
 
-Depth Of Field includes [more options](More-Options.html) that you must manually expose.
-
+Depth Of Field includes properties that you define in the quality presets (**Project Settings > Quality > HDRP > Post-processing quality settings**). You can override these properties locally when using the Custom quality option. In this case, more options will appear in the UI.   
```

Comment:
**Depth Of Field** includes properties which you can define in your Project's quality presets (**Project Settings > Quality > HDRP > Post-processing quality settings**). If you do not want to use a quality preset and instead want to override these properties locally, select **Custom** from the **Quality** drop-down. This makes more properties appear in the UI which you can then edit.

Reply:
Thanks Lewis. I will keep this PR on hold for awhile until we have a meeting with UX, and then integrate your changes in the final version.